static/js/compose_state.js

```diff
@@ -36,9 +36,18 @@ function get_or_set(fieldname, keep_leading_whitespace) {
     };
 }
 
-// TODO: Break out setters and getter into their own functions.
-export const stream_name = get_or_set("stream_message_recipient_stream");
+export function stream_name() {
+    return $("#stream_message_recipient_stream").val().trim();
+}
```

Comment:
It would have been nice for the commit message to mention that this is equivalent to the previous code because the `keep_leading_whitespace` parameter is never used with this function -- as it was, probably half the time reviewing this PR was reading the tangle that is `get_or_set` and figuring out that's the reason one should expect no functional changes here.

(I added such a comment while merging as an example)